website/src/documentation/sdks/java-dependencies.md

```diff
@@ -91,7 +91,7 @@ the relevant section below.</p>
   <tr><td>io.grpc</td><td>grpc-stub</td><td>1.13.1</td></tr>
   <tr><td>com.google.guava</td><td>guava</td><td>20.0</td></tr>
   <tr><td>com.google.guava</td><td>guava-testlib</td><td>20.0</td></tr>
-  <tr><td>org.apache.hadoop</td><td>hadoop-client</td><td>2.7.3</td></tr>
```

Comment:
This file lists dependency versions at specific SDK versions and isn't meant to be edited when versions change at head.